tests/Type/PhpEnumType/MyCustomPhpEnum.php

```diff
@@ -6,7 +6,7 @@
 use GraphQL\Type\Definition\Description;
 
 #[Description(description: 'foo')]
-enum PhpEnum
+enum MyCustomPhpEnum
```

Comment:
This doesn't have anything to do with the feature, it just always vaguely confuses me when sample code uses variable names that sound like language features or the name of the thing being demonstrated, eg:

```php
$float = 5;
```

Reply:
Makes sense to me.